docs/modules/ROOT/pages/websockets.adoc

```diff
@@ -4,20 +4,20 @@ include::./includes/attributes.adoc[]
 
 Using a chatbot in a WebSockets environment is quite common, which is why the extension provides a few facilities to make such usages as easy as possible.
 
-* 1. Start by adding the link:https://quarkus.io/guides/websockets-next-tutorial[quarkus-websockets-next] dependency to your `pom.xml` file:
-
+. Start by adding the link:https://quarkus.io/guides/websockets-next-tutorial[quarkus-websockets-next] dependency to your `pom.xml` file:
++
 [source,xml]
 ----
 <dependency>
     <groupId>io.quarkus</groupId>
     <artifactId>quarkus-websockets-next</artifactId>
 </dependency>
 ----
-
++
 IMPORTANT: `quarkus-websockets-next` is available as of Quarkus 3.9.
 
-* 2. Annotated your AiService with `@SessionScoped`
-
+. Annotate your AI service with `@SessionScoped`
```

Comment:
`Annotate` was actually what I wanted to fix.
As for `AI service`, I made it consistent with the notes at the bottom.